src/style.css

```diff
@@ -1,3 +1,76 @@
 body {
+  --selected-color: #00acdc;
+
   margin: 0;
+  padding: 0;
+  font-weight: 500;
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.logoLink {
+  display: flex;
+  align-items: center;
```

Comment:
display flex is unnecessary here, all you need is to add width and height to `<a>` element ( it could have class `header__logo`)